kociemba/pykociemba/tools.py

```diff
@@ -54,3 +54,23 @@ def randomCube():
             break
     fc = cc.toFaceCube()
     return fc.to_String()
+
+def randomLLCube():
```

Comment:
I guess the name might be more self-explaining, say `randomLastLayerCube`, what do you think?